core/dropdowndiv.js

```diff
@@ -744,5 +744,11 @@ DropDownDiv.repositionForWindowResize = function() {
     DropDownDiv.hide();
   }
 };
-
 exports = DropDownDiv;
+
+exports.testOnly_setGetBoundsInfo = function(getBoundsInfoMock) {
+  goog.setTestOnly();
+  getBoundsInfo = getBoundsInfoMock;
+};
+
+exports.testOnly_getPositionMetrics = getPositionMetrics;
```

Comment:
Pedantically, this should probably be a wrapper:

```JS
/** ... */
exports.testOnly_getPositionMetrics = function (/* ... */) {
  goog.setTestOnly();
  return getPositionMetrics(arguments);
}
```

Since we may have many such wrappers it might be worth defining a higher-order function to create them, so you could just write something like:

```JS
exports.testOnly_foo = testUtils.testOnly(foo);
```